### ISSUE TITLE
[A11Y] Image library view is not keyboard accessible

### ISSUE THREAD

**Issue by laryn** [backdrop/backdrop-issues#6360]
## Description of the bug

I was just testing something out in the image library modal in an editor window, and I realized the image library itself is not accessible for keyboard users. Tabbing through just skips over all the images completely.

**Comment by indigoxela** [comment]
The question is, how this is supposed to work.

Usually image files get no tab index - that's normal behavior. But in this case images should be an interactive thing, like a form item or so.

I have to admit, that I'm not familiar enough with A11Y in image item selection, to be able to suggest a solution.

**Comment by laryn** [comment]
If the images had an `<a>` element around them, that would get focus, but the library is built just with `<div>`s and javascript, I think. When an image is selected, the surrounding div gets a class:

```
<div class="image-library-choose-file image-library-image-selected">
```

I wonder if we can follow the pattern we have on Layout template selection:
![CleanShot 2024-01-11 at 08 31 32@2x](https://github.com/backdrop/backdrop-issues/assets/871421/1c1007d8-d2a5-4304-89d6-9bf1ea5766d5)

So you'd tab into the library, left/right arrows would go between images (autoselecting them) and then another tab would take you to the pager or insert button.

**Comment by laryn** [comment]
This article has a section on using the keyboard with Drupal's media library:
https://imagexmedia.com/blog/keyboard-accessibilty-in-drupal

**Comment by indigoxela** [comment]
> If the images had an <a> element around them, that would get focus

Yes, or if we could just add the attribute `tabindex="0"`. Should we?

**Comment by laryn** [comment]
It's worth testing!
